cmd/dockerd/daemon.go

```diff
@@ -450,23 +450,21 @@ func newRouterOptions(ctx context.Context, config *config.Config, d *daemon.Daem
 			Named:    d.ImageNamedByBuildkit,
 		},
 	})
-	if err != nil {
-		return routerOptions{}, err
-	}
 
 	bb, err := buildbackend.NewBackend(d.ImageService(), manager, bk, d.EventsService)
 	if err != nil {
```

Comment:
On a fun note; it looks like `NewBackend` never return an error; not for this PR, but wondering if we should even get rid of the constructor (or remove the `error` return) https://github.com/moby/moby/blob/2f6953fcd8a411816ea84f4d24fd9ea101c19ba7/api/server/backend/build/backend.go#L40-L43